src/content/docs/dropins/all/experimenting.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Experimenting with drop-in components
+description: Learn about how to create A/B experiments for drop-in components.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import Diagram from '@components/Diagram.astro';
+import Vocabulary from '@components/Vocabulary.astro';
+import Aside from '@components/Aside.astro';
+import Callouts from '@components/Callouts.astro';
+import { Steps } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+
+This topic introduces you to the concept of creating A/B experiments for drop-in components. You will learn how to create experiments, define variants, and track the performance of each variant.
+
+## Vocabulary
+
+<Vocabulary>
+
+### A/B experiment
+
+A test that compares two or more pages to determine which page performs better.
+
+### Variant
+
+A version of a page that is different from the original version. Variants are created to test different features, designs, or content.
+
+### Control
+
+The original version of the drop-in component that is used as a baseline for comparison.
+
+### Challenger
+
+The variant of the drop-in component being tested.
+
+</Vocabulary>
+
+## Step-by-step
+
+The steps for creating an A/B experiment for a commerce storefront page, are summarized below, and detailed in the following tasks:
+
+<Steps>
+1. Create a variant page for the experiment.
+1. Add the experiment to the metadata block of the original page.
+1. Preview the experiment using AEM Sidekick Preview button.
+1. Make code changes for experiments, if necessary.
+</Steps>
+
+<Tasks>
+<Task>
+### Create an variant page for the experiment
+
+Duplicate the original page and make the necessary changes to create a new variant page for the experiment. Update any placeholder labels or content in the variant page to reflect the changes you want to test. For example, you can change the color of a button, update the text of a call-to-action, or rearrange the layout of the page. Save the variant page with a new name to distinguish it from the original page.
+
+</Task>
+
+<Task>
+### Add the experiment to the original page
+
+Add the experiment to the metadata block of the original page by adding the experiment name and the URL to the experiment page. For example:
```

Comment:
I'm assuming that this is intentionally vague, but it would be good to follow up with some specific examples of experimentation. There was at least one example in the GA demo, though I don't recall specifically what it was.

Reply:
On it!